zirgen/dsl/Library/src/bits.zir

```diff
@@ -19,17 +18,6 @@ component BitReg(val: Val) {
   reg
 }
 
-// Check that valid bits are valid
```

Comment:
what was the reasoning for moving the tests out of here into a separate file? Is there some technical reason that has to happen, or was it a style choice?